### ISSUE TITLE
[Bug]: New UI: Stackoverflow

### ISSUE THREAD

**Issue by xome** [mallowigi/material-theme-issues#286]
### Describe the bug

IntelliJ 2022.3 with new UI enabled raises Exceptions. Stacktrace is attached.
There is no visible consequence except for the notification.

### To Reproduce

Start IntelliJ with new UI enabled. Wait for Notifications to pop up

### Expected behavior

No notifications are raised

### Material Theme Version

v8.3.2

### IDE

IntelliJ IDEA

### System Info

```shell
IntelliJ IDEA 2022.3 (Ultimate Edition)
Build #IU-223.7571.182, built on November 29, 2022
Licensed to xxxx
Subscription is active until xxxxx
Runtime version: 17.0.5+1-b653.14 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 10 10.0
GC: G1 Young Generation, G1 Old Generation
Memory: 6000M
Cores: 16
Registry:
    editor.focus.mode.color.light=6a737d
    ide.editor.tab.selection.animation=true
    debugger.new.tool.window.layout=true
    ide.new.editor.tabs.vertical.borders=true
    ide.experimental.ui=true
    ide.balloon.shadow.size=0
    editor.focus.mode.color.dark=6a737d
    database.legacy.navigate.to.code.from.tree=true

Non-Bundled Plugins:
    org.logtalk (1.0.15)
    com.thvardhan.gradianto (4.5)
    com.perl5 (2022.3)
    com.intellij.ideolog (203.0.30.0)
    CMD Support (1.0.5)
    org.intellij.RegexpTester (1.0.8)
    tech.phosphorus.intellij-prolog (0.1.3.0-beta3)
    net.seesharpsoft.intellij.plugins.csv (3.0.1-223)
    cucumber-javascript (223.7571.113)
    com.haulmont.jpab (2022.5.1-223)
    com.chrisrm.idea.MaterialThemeUI (8.3.2)

Kotlin: 223-1.7.20-release-201-IJ7571.182
```

### Logs/Screenshots

```shell
java.lang.StackOverflowError
	at java.base/java.util.concurrent.ConcurrentHashMap.computeIfAbsent(ConcurrentHashMap.java:1694)
	at com.intellij.openapi.util.registry.Registry.doGet(Registry.java:59)
	at com.intellij.openapi.util.registry.Registry._getWithoutStateCheck(Registry.java:55)
	at com.intellij.openapi.util.registry.RegistryManagerImpl.is(RegistryManagerImpl.kt:51)
	at com.intellij.openapi.util.registry.EarlyAccessRegistryManager.getBoolean(EarlyAccessRegistryManager.kt:81)
	at com.intellij.ui.ExperimentalUI.isNewUI(ExperimentalUI.java:43)
	at com.mallowigi.idea.projectframe.MTTitleFrame.getUsesColoredTitleBar(MTTitleFrame.kt:59)
	at com.mallowigi.idea.projectframe.MTTitleFrame.setTitleBarColor(MTTitleFrame.kt:143)
	at com.mallowigi.idea.projectframe.MTTitleFrame.setTitleBarColor(MTTitleFrame.kt:145)
	at com.mallowigi.idea.projectframe.MTTitleFrame.access$setTitleBarColor(MTTitleFrame.kt:47)
	at com.mallowigi.idea.projectframe.MTTitleFrame$addBackgroundChangeListener$listener$1.invoke(MTTitleFrame.kt:130)
	at com.mallowigi.idea.projectframe.MTTitleFrame$addBackgroundChangeListener$listener$1.invoke(MTTitleFrame.kt:130)
	at com.mallowigi.idea.projectframe.MTTitleFrame.addBackgroundChangeListener$lambda$6(MTTitleFrame.kt:136)
	at java.desktop/java.beans.PropertyChangeSupport.fire(PropertyChangeSupport.java:343)
	at java.desktop/java.beans.PropertyChangeSupport.firePropertyChange(PropertyChangeSupport.java:336)
	at java.desktop/java.beans.PropertyChangeSupport.firePropertyChange(PropertyChangeSupport.java:268)
	at java.desktop/java.awt.Component.firePropertyChange(Component.java:8750)
	at java.desktop/java.awt.Component.setBackground(Component.java:1909)
	at java.desktop/javax.swing.JComponent.setBackground(JComponent.java:2800)
	at com.mallowigi.idea.projectframe.MTTitleFrame.setTitleBarColor(MTTitleFrame.kt:142)
	at com.mallowigi.idea.projectframe.MTTitleFrame.access$setTitleBarColor(MTTitleFrame.kt:47)
	at com.mallowigi.idea.projectframe.MTTitleFrame$addBackgroundChangeListener$listener$1.invoke(MTTitleFrame.kt:130)
	at com.mallowigi.idea.projectframe.MTTitleFrame$addBackgroundChangeListener$listener$1.invoke(MTTitleFrame.kt:130)
	at com.mallowigi.idea.projectframe.MTTitleFrame.addBackgroundChangeListener$lambda$6(MTTitleFrame.kt:136)
	
	[...] exempted because repetition
	
	at java.desktop/java.beans.PropertyChangeSupport.fire(PropertyChangeSupport.java:343)
	at java.desktop/java.beans.PropertyChangeSupport.firePropertyChange(PropertyChangeSupport.java:336)
	at java.desktop/java.beans.PropertyChangeSupport.firePropertyChange(PropertyChangeSupport.java:268)
	at java.desktop/java.awt.Component.firePropertyChange(Component.java:8750)
	at java.desktop/java.awt.Component.setBackground(Component.java:1909)
	at java.desktop/javax.swing.JComponent.setBackground(JComponent.java:2800)
	at com.mallowigi.idea.projectframe.MTTitleFrame.setTitleBarColor(MTTitleFrame.kt:142)
	at com.mallowigi.idea.projectframe.MTTitleFrame.access$setTitleBarColor(MTTitleFrame.kt:47)
	at com.mallowigi.idea.projectframe.MTTitleFrame$addBackgroundChangeListener$listener$1.invoke(MTTitleFrame.kt:130)
	at com.mallowigi.idea.projectframe.MTTitleFrame$addBackgroundChangeListener$listener$1.invoke(MTTitleFrame.kt:130)
```

**Comment by Evert-Arends** [comment]
I'm experiencing this aswel, I can't seem to kill Focus Mode.

**Comment by mallowigi** [comment]
I can't reproduce on Mac. I'll check on Windows whenever I can get my hands on a Windows machine.

**Comment by Evert-Arends** [comment]
I have a linux desktop where is issue arrises, can I provide some info for you?

**Comment by Evert-Arends** [comment]
I notice that we both are running the same java version from Jetbrains?
```
Runtime version: 17.0.4.1+7-b469.62 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
```

**Comment by Evert-Arends** [comment]
It happens when I hit "focus mode" in the plugin settings. I can't seem to turn it off and I really loathe it. I just wanted to try but it's not for me.

**Comment by xome** [comment]
Hi,
can't reproduce it on Windows 11. I'll try to figure out what triggers it on my Win10 machine.

**Comment by xome** [comment]
The issue seems to be fixed in 2022.3.1 (for Windows 10 at least).
If you're happy with that, you can close the issue imo.

Greetings
